server/models/index.js

```diff
@@ -1,14 +1,16 @@
+/* eslint-disable no-console */
```

Comment:
consider adding to your eslint configuration a rule to disable no-console so you don't need to add this rule to each individual file: https://eslint.org/docs/rules/no-console